### PR TITLE
Add Geist to Aave v2 forks

### PR DESCRIPTION
**Pull request by dmelotik**
NOTICE: 
- Should be merged after #427 

Deliverables:
- Add Geist to Aave v2 forks

Watching:
- Make sure GEIST price is correct for reward purposes

TODO:
- [x] Get rewards working
- [x] figure out how to get token supplys and prices to pass into 
- [x] `_handleReserveDataUpdated()`

### CODE REVIEW

**Comment by this-username-is-taken** [comment]
can you rebase onto `origin/master`?

**Comment by dmelotik** [comment]
done @this-username-is-taken

**Comment by this-username-is-taken** [comment]
you probably want to rebase and force push

File: deployment/deploymentConfigurations.json [hunk 1]
```diff
@@ -72,6 +72,14 @@
           "dmelotik": "dmelotik/aave-v2-rwa",
           "deploy-on-merge": true
         }
+      },
+      "geist": {
+        "fantom": {
+          "template": "geist.template.yaml",
+          "messari": "messari/geist-fantom",
```
Comment (this-username-is-taken) [inline deployment/deploymentConfigurations.json:79]: our subgraph is call `geist-finance-fantom`:
https://thegraph.com/hosted-service/subgraph/messari/geist-finance-fantom

File: subgraphs/aave-v2-forks/protocols/geist/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+`Borrows`
+
+`Liquidations`
+
+`Repayments`
+
+### Reward Token Emissions Amount
+
+To be added
```
Comment (this-username-is-taken) [inline subgraphs/aave-v2-forks/protocols/geist/README.md:69]: might want to update this

File: subgraphs/aave-v2-forks/protocols/geist/src/constants.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "0x668AE94D0870230AC007a01B471D02b2c94DDcB9";
+export const GFTM_ADDRESS = "0x39b3bd37208cbade74d0fcbdbb12d606295b430a";
+
+/////////////////////////////
+///// Protocol Specific /////
+/////////////////////////////
+
+export namespace Protocol {
+  export const NAME = "Geist";
+  export const SLUG = "geist";
```
Comment (this-username-is-taken) [inline subgraphs/aave-v2-forks/protocols/geist/src/constants.ts:19]: slug should align with coingecko, which is `geist-finance`

File: subgraphs/aave-v2-forks/protocols/geist/src/constants.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+export const GEIST_FTM_LP_ADDRESS =
+  "0x668AE94D0870230AC007a01B471D02b2c94DDcB9";
+export const GFTM_ADDRESS = "0x39b3bd37208cbade74d0fcbdbb12d606295b430a";
+
+/////////////////////////////
+///// Protocol Specific /////
+/////////////////////////////
+
+export namespace Protocol {
+  export const NAME = "Geist";
```
Comment (this-username-is-taken) [inline subgraphs/aave-v2-forks/protocols/geist/src/constants.ts:18]: also change to `Geist Finance`

**Comment by dmelotik** [comment]
Messed up the rebase, closing and making a new PR, the changes suggested by @this-username-is-taken are fixed

**Pull request closed without merging**